apps/src/templates/instructions/codeReview/CommentEditor.jsx

```diff
@@ -23,35 +62,38 @@ export default class CommentEditor extends Component {
           onChange={this.commentChanged}
           value={this.state.comment}
         />
-        <div
-          style={{
-            ...styles.buttonContainer
-          }}
-        >
-          {comment && (
-            <Button
-              key="cancel"
-              text={msg.cancel()}
-              onClick={() => this.setState({comment: ''})}
-              color="gray"
-              style={{...styles.buttons.all, ...styles.buttons.cancel}}
-            />
-          )}
-          {comment && (
-            <Button
-              key="submit"
-              text={msg.submit()}
-              onClick={() => this.props.onNewCommentSubmit(comment)}
-              color="orange"
-              style={{...styles.buttons.all, ...styles.buttons.submit}}
-            />
-          )}
+        <div style={styles.commentFooter}>
+          {comment && this.renderSaveStatus()}
```

Comment:
Nit: maybe just include this if we have a save in progress or save error?

Reply:
we actually want the empty div in this case to keep the buttons over to the right